docs/source/resource_files.rst

```diff
@@ -23,21 +23,34 @@ The following paths are checked in order:
 
 - For transformation grids that have an explict relative or absolute path,
   the directory specified in the grid filename.
+
 - Path resolved by the callback function set with
   the :c:func:`proj_context_set_file_finder`. If it is set, the next tests
   will not be run.
+
 - Path(s) set with the :c:func:`proj_context_set_search_paths`. If set, the
   next tests will not be run.
+
+- The PROJ user writable directory, which is :
+
+    * on Windows, ${LOCALAPPDATA}/proj
+    * on MacOSX, ${HOME}/Library/Logs/proj
```

Comment:
Is this really the correct plase to put files on OS X? On my OS X installation I see that QGIS has put files in `~/Library/Application Support/` and `~/Library/Caches/`. Both sounds like better places for us to put data. I couldn't easily find an overview of what is intended to go in the subfolders of `~/Library/`

Reply:
You're right. I was pointed during RFC-4 review to https://github.com/ActiveState/appdirs/blob/a54ea98feed0a7593475b94de3a359e9e1fe8fdb/appdirs.py#L45-L97 which indeed points to ~/Library/Application Support/, but in this file there was also used of ~/Library/Logs and I erroneously used that. I'll change that to ~/Library/Application Support/ which makes more sense

Reply:
Fixed per 9d2851ad362a32e756986be65e04914230bd9ab5

Reply:
and doc update per 9bb87ea4782b6005848afde29db9751e05ee0e55

Reply:
Thanks